src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/services/StudyService.java

```diff
@@ -18,18 +18,30 @@
 package uk.ac.ebi.ampt2d.metadata.persistence.services;
 
 import com.querydsl.core.types.Predicate;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.AccessionVersionEntityId;
 import uk.ac.ebi.ampt2d.metadata.persistence.entities.Study;
 
+import java.time.LocalDate;
 import java.util.List;
 
 public interface StudyService {
```

Comment:
Do we really need a separate interface?

Reply:
No, we don't really need an interface. But having one here is more consistent with Spring framework among other things. Since it is already here I will stick with it.